_posts/en/newsletters/2021-07-28-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #159'
+permalink: /en/newsletters/2021/07/28/
+name: 2021-07-28-newsletter
+slug: 2021-07-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes our regular sections with the best
+questions and answers of the past month from the Bitcoin Stack Exchange,
+our latest column about preparing for taproot, a list of new software
+releases and release candidates, and descriptions of notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+*No significant news this week.*
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
```

Comment:
Would it perhaps make sense to prescribe this alias site-wide?

Reply:
I think it's @bitschmidty's call, since he's using this.  It would change the variable name from `{{bse}}` to `{{site.bse}}` (or `site.whatever_mike_wants`), which might be a tiny bit more effort to type.